src/ffjavascript.d.ts

```diff
@@ -0,0 +1,57 @@
+// Type definitions for the ffjavascript library. Types are not complete and primarily match our use of the library
+declare module "ffjavascript" {
```

Comment:
💯 
Might be even worth it extrapolating this later on into its own package or stg and ping the iden3/circom guys